### ISSUE TITLE
Provide an early list of alpha connectors that are likely to be Cloud eligible 

### ISSUE THREAD

**Issue by alafanechere** [airbytehq/airbyte#21648]
Spike a QA engine prototype to provide the product team with a list of potential eligible connectors as of today. The hardest requirements to check can be ignored. This step aims to ensure the _Deploy All Eligible Connectors to cloud_ is a high-leverage one.

The code of the prototype won't be shared/reviewed. 
We only need a google sheet to share with the product team a list all the connectors that are matching most of the requirements we defined for a connector to be a cloud:

- type
- name
- version
-  is already on cloud	
- icon is available	
- latest build is successful	
- documentation is available	
- number of connections
- number of users	
- sync success rate
- changelog is updated

**Comment by alafanechere** [comment]
This [Google sheets](https://docs.google.com/spreadsheets/d/1J7bCnTcme-_F0nNDU2SspPymSa8K2Ma1utwh6ZpewKE/edit#gid=0) provides the requested information.

**TL;DR**: 
- We have 174 connectors not deployed to Cloud
- **46 are likely immediately eligible for deployment to Cloud**: 

|Type       |Name          |
|-----------|--------------|
|destination|Kafka         |
|source     |Kafka         |
|source     |Mailgun       |
|destination|S3 Glue       |
|source     |Elasticsearch |
|source     |Zoom          |
|source     |xkcd          |
|source     |Auth0         |
|source     |Cart.com      |
|source     |OpenWeather   |
|destination|Kinesis       |
|destination|YugabyteDB    |
|source     |LaunchDarkly  |
|source     |New York Times|
|source     |Sendinblue    |
|source     |Senseforce    |
|destination|MQTT          |
|source     |GetLago       |
|source     |Glassfrog     |
|source     |Looker        |
|source     |Weatherstack  |
|source     |Coda          |
|source     |EmailOctopus  |
|source     |Klarna        |
|source     |Omnisend      |
|source     |Cockroachdb   |
|source     |ConfigCat     |
|source     |Datascope     |
|source     |Freshcaller   |
|source     |Gridly        |
|source     |GNews         |
|source     |Hubplanner    |
|source     |IP2Whois      |
|source     |IBM Db2       |
|source     |Insightly     |
|source     |Lokalise      |
|source     |Pocket        |
|source     |PyPI          |
|source     |Secoda        |
|source     |Smaily        |
|source     |SmartEngage   |
|source     |Statuspage    |
|source     |TiDB          |
|source     |Vantage       |
|source     |Younium       |
